### PR TITLE
Change CCDA Test URL from personal repository to Continuous Delivery …

### LINKED ISSUE #39: Change CCDA Tests URL
Change CCDA Test URL from personal repository to Continuous Delivery one.

### PR DESCRIPTION
**Pull request by sgroh**
Change CCDA Test URL from personal repository to Continuous Delivery one.

### CODE REVIEW

File: leap-ces-ccda-orchestration/src/test/java/gov/hhs/onc/leap/ces/ccda/orchestration/service/CCDAOrchestrationServiceTest.java [hunk 1]
```diff
@@ -58,15 +57,7 @@ public void processCCDADocumentCardTest1() throws Exception {
...
-
-        ObjectMapper mapper = new ObjectMapper();
-        String request = mapper.writeValueAsString(hookRequestWithData);
-        //writeRequestToFile(request, "CCDACardTestWithDcoument.json");
-
-        String result = makeRequest(request);
-
-        //System.out.println(result);
-
+        String result = doRequest(hookRequestWithData);
```
Comment (mojitoj) [inline leap-ces-ccda-orchestration/src/test/java/gov/hhs/onc/leap/ces/ccda/orchestration/service/CCDAOrchestrationServiceTest.java:60]: This should be moved to a config file.

File: leap-ces-ccda-orchestration/src/test/java/gov/hhs/onc/leap/ces/ccda/orchestration/service/CCDAOrchestrationServiceTest.java [hunk 2]
```diff
@@ -29,9 +29,8 @@
 public class CCDAOrchestrationServiceTest {
     private static final Logger LOGGER = Logger.getLogger(CCDAOrchestrationServiceTest.class.getName());
     //orchestration endpoint
-    private String host =  "http://34.94.253.50:9093";
+    private String host =  "https://sdhc-leap-ccda-axz2tb4tma-uc.a.run.app";
```
Comment (mojitoj) [inline leap-ces-ccda-orchestration/src/test/java/gov/hhs/onc/leap/ces/ccda/orchestration/service/CCDAOrchestrationServiceTest.java]: This should be moved to a config file or env variable. I don't think hardcoding it is a good idea.

**Review by mojitoj — CHANGES_REQUESTED** [review]

File: leap-ces-ccda-orchestration/src/test/java/gov/hhs/onc/leap/ces/ccda/orchestration/service/CCDAOrchestrationServiceTest.java (hunk 2 above)
Comment (sgroh) [reply leap-ces-ccda-orchestration/src/test/java/gov/hhs/onc/leap/ces/ccda/orchestration/service/CCDAOrchestrationServiceTest.java]: done.

File: leap-ces-ccda-orchestration/src/test/resources/application-test.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+ccda:
+  host:
+    url: https://sdhc-leap-ccda-axz2tb4tma-uc.a.run.app
+
+cds:
```
Comment (mojitoj) [inline leap-ces-ccda-orchestration/src/test/resources/application-test.yaml:5]: I don't see where these are used?

**Review by mojitoj — APPROVED** [review]

File: leap-ces-ccda-orchestration/src/test/resources/application-test.yaml (hunk 3 above)
Comment (sgroh) [reply leap-ces-ccda-orchestration/src/test/resources/application-test.yaml:5]: This is used when you annotate a test with @SpringBootApplication, this allows to load application context configuration and therefore allows us to use @Value on tests. Without this the test will fails creating the context for the respective service  `CCDAOrchestrationService`.

**Pull request merged**